src/test/regress/expected/distributed_planning.out

```diff
@@ -2,12 +2,36 @@ SET search_path TO "distributed planning";
 -- Confirm the basics work
 INSERT INTO test VALUES (1, 2), (3, 4), (5, 6), (2, 7), (4, 5);
 INSERT INTO test VALUES (6, 7);
+-- Insert two edge case values, the first value hashes to MAX_INT32 and the
```

Comment:
This seems to be changing a lot of other stuff. Do you think it makes sense to add these test to `adaptive_executor_repartition_sql`. We can add 5-6 lines in total for this specific case (along with the create statement for a new table maybe).

Also would be good to add a comment like:

```
-- this test will probably become invalid if we change the number of worker nodes in the regression test
-- or if we change the value of `citus.repartition_join_bucket_count_per_node`.
```

Reply:
This doesn't just matter for repartion joins, but also for regular queries (once https://github.com/citusdata/citus/pull/5418 is merged). That's why I put it here in the core data. This way it would also be tested with different numbers of nodes (which mattered for the issue we ran into). 

I realise it changes some output, but they are all changed in expected ways. So I think it's worth it to have this edge case tested in more queries and setups than just the one where we had issues.

Reply:
Yeah that's a valid point, and I think my other comment is not incorrect as shard or partition count or the node count is irrevelant with the current testing as long as it is not a power of 2.